perception/sub8_perception/nodes/path_marker_finder.py

```diff
@@ -37,6 +38,11 @@ class PathMarkerFinder():
     # Model of four corners of path marker, centered around 0 in meters
     LENGTH = 0.6096 # Longer side of path marker in meters
     WIDTH = 0.0762 # Shorter Side
+
+    # Demensions of the prop we made for testing
```

Comment:
I see, what you mean. Although you could just add a `/path_marker/set_geometry` service, and make sure you print out what shape you're looking for.